src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/route_config/AuthenticationRouteConfig.kt

```diff
@@ -14,6 +14,8 @@ object AuthenticationRouteConfig : RouteConfig
                     .basicAuth(),
             Endpoint("/set-shiny-cookie/", controller, "setShinyCookie")
                     .secure()
+                    .json(),
+            Endpoint("/clear-shiny-cookie/", controller, "clearShinyCookie")
```

Comment:
I think you're right that this doesn't need to be `secure`, but given that it's a bit unusual in the API, maybe add a comment here explaining why, so it doesn't surprise/confuse us in the future?

Reply:
yes good idea